operations/observability/mixins/meta/rules/components/db-sync/alerts.libsonnet

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Licensed under the MIT License. See License-MIT.txt in the project root for license information.
+ */
+
+{
+  prometheusAlerts+:: {
```

Comment:
I think you need to add it [here](https://github.com/gitpod-io/gitpod/blob/a62a63e87d09724da301f1f75d5c22e420edd2ca/operations/observability/mixins/meta/rules/components/components.libsonnet).